### ISSUE TITLE
Load history form should provide keyboard access to grid

### ISSUE THREAD

**Issue by TamarEGranor** [VFPX/GoFish#10]
## 📝 Provide a description of the new feature

There's no way to access the grid in the Load History function without a mouse. There should be, for accessibility reasons.

_What is the expected behavior of the proposed feature?  What is the scenario this would be used?_

The grid should be in the tab order of the form, or accessible via a special keystroke (noted on the form).

---

If you'd like to see this feature implemented, add a 👍 reaction to this post.

**Comment by DougHennig** [comment]
If you figure out what change to make, I'll implement it.

**Comment by lscheffler** [comment]
The columns and the textboxes of the columns of the grid are, why ever, READONLY. This seems to prevent to enter using tab. I change this and set tab order so, that the grid has focus on start.

Something learned ...

**Comment by lscheffler** [comment]
Added CR to grids keypress to open from kbd
